WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -34,31 +34,35 @@ struct POSSendReceiptView: View {
                     }
                 }))
 
-                VStack(alignment: .center, spacing: conditionalPadding(62)) {
-                    TextField("",
-                              text: $textFieldInput,
-                              prompt: Text(Localization.textfieldPlaceholder).foregroundColor(.posOnDisabledContainer))
-                    .foregroundStyle(Color.posOnSurface)
-                    .dynamicTypeSize(...DynamicTypeSize.accessibility1)
-                    .keyboardType(.emailAddress)
-                    .textInputAutocapitalization(.never)
-                    .autocorrectionDisabled()
-                    .multilineTextAlignment(.center)
-                    .font(POSFontStyle.posHeadingRegular)
-                    .focused()
-                    .focused($isTextFieldFocused)
-                    .padding()
-                    .onSubmit {
-                        sendReceipt()
-                    }
+                VStack(alignment: .center, spacing: conditionalPadding(POSSpacing.medium)) {
+                    Spacer()
 
-                    if let errorMessage = errorMessage {
-                        Text(errorMessage)
-                            .font(POSFontStyle.posBodySmallRegular())
-                            .foregroundColor(.posError)
-                            .padding(.bottom, Constants.errorMessagePadding)
+                    VStack(alignment: .center, spacing: POSSpacing.xSmall) {
```

Comment:
I've moved the text field + optional error message to a VStack with xSmall spacing as in the cash view.